doc/user/acl.md

```diff
@@ -0,0 +1,150 @@
+# DAOS Access Control
+
+Client access to DAOS resources like pools and containers is controlled by
+Access Control Lists (ACLs). A DAOS ACL is a list of zero or more Access Control
+Entries (ACEs). ACEs are the individual rules applied to each access decision.
+
+## Access Control Entries
+
+In the input and output of DAOS tools, an ACE is defined using a colon-separated
+string format:\
+`TYPE:FLAGS:IDENTITY:PERMISSIONS`
+
+The contents of all the fields are case-sensitive.
+
+### Type
+
+The type of entry. Only one type of ACE is supported at this time.
+
+* A (Allow): Allow access to the specified identity for the given permissions.
+
+### Flags
+
+The flags provide additional information about how the ACE should be
+interpreted.
+
+* G (Group): The identity should be interpreted as a group.
+
+### Identity
+
+The identity (also called the principal) is specified in the name@domain format.
+The domain should be left off if the name is a UNIX user/group on the local
+domain. Currently, this is the only case supported by DAOS.
+
+There are three special identities, `OWNER@`, `GROUP@` and `EVERYONE@`,
+which align with User, Group, and Other from traditional POSIX permission bits.
+When providing them in the ACE string format, they must be spelled exactly as
+written here, in uppercase with no domain appended. The `GROUP@` entry must
+also have the `G` (group) flag.
+
+### Permissions
+
+The permissions in a resource's ACL permit a certain type of user access to
+the resource. Which types of permissions are valid depends on the resource
+type.
+
+| Permission	| Pool Meaning		| Container Meaning		|
+| ------------- | --------------------- | ----------------------------- |
+| r (Read)	| Alias for 't'		| Read data and attributes	|
+| w (Write)	| Alias for 'c' + 'd'	| Write data and attributes	|
+| c (Create)	| Create containers	| N/A				|
+| d (Delete)	| Delete any container	| Delete this container		|
+| t (Get-Prop)	| Connect/query		| Get container properties	|
+| T (Set-Prop)	| N/A			| Change container properties	|
+| a (Get-ACL)	| N/A			| Get container ACL		|
+| A (Set-ACL)	| N/A			| Change container ACL		|
+| o (Set-Owner)	| N/A			| Change owner user and group	|
+
+ACLs containing permissions not applicable to the given resource are considered
+invalid.
+
+In general a user/group's permissions must include at least some form of
```

Comment:
Suggest reordering the presentation of the ideas in the sentence to avoid potential confusion with the next section Denying Access. The confusion is that this sentence starts out saying in general a user/group perms must have at least read access. And if you don't read carefully enough the rest, then move ahead to the Denying Access section the text there seems to conflict with the idea expressed here.

How about "To allow a user/group to connect to a resource, that principal's permissions must include at least some form of read access (for example, read or get-prop)."

Reply:
Good point - I've simplified it a bit in the next revision, which I hope reads more clearly.